### PR TITLE
QOLSVC-2117 CHDEXCHAE updates

### PR DESCRIPTION
**Pull request by duttonw**
Consolidation of Linked Digital work:
* QOLSVC-2041 – CHDEXCHAE-62
  Branch: https://github.com/DataShades/ckanext-data-qld/tree/CHDEXCHAE-62
  PR: https://github.com/DataShades/ckanext-data-qld/pull/3
* QOLSVC-2038 – CHDEXCHAE-57
  No changes to theme.
  PR: https://github.com/qld-gov-au/ckanext-validation/pull/46
*  QOLSVC-2039 – CHDEXCHAE-61
  Branch: https://github.com/DataShades/ckanext-data-qld/tree/CHDEXCHAE-61
  PR: https://github.com/qld-gov-au/ckanext-resource-visibility/pull/10
  PR: https://github.com/DataShades/ckanext-data-qld/pull/4
*  QOLSVC-2040 – CHDEXCHAE-60
  Branch: https://github.com/DataShades/ckanext-data-qld/tree/CHDEXCHAE-60
  PR: https://github.com/qld-gov-au/ckanext-resource-visibility/pull/11 
  PR: https://github.com/DataShades/ckanext-data-qld/pull/5
* QOLSVC-2077 – CHDEXCHAE-59
  Branch: https://github.com/DataShades/ckanext-data-qld/tree/CHDEXCHAE-59
  PR: https://github.com/DataShades/ckanext-data-qld/pull/6  
  PR: https://github.com/qld-gov-au/ckanext-resource-visibility/pull/12
*  QOLSVC-2078 – CHDEXCHAE-67
  Branch: https://github.com/DataShades/ckanext-data-qld/tree/CHDEXCHAE-67
  PR: https://github.com/DataShades/ckanext-data-qld/pull/7 
  PR: https://github.com/qld-gov-au/ckanext-resource-visibility/pull/14

### CODE REVIEW

**Review by ThrawnCA — APPROVED** [review]

**Comment by ThrawnCA** [comment]
Two tests are failing. There's an outdated assertion that still expects the "Field 'foo' not in data" message, and a situation where "HIDDEN" isn't displaying when expected.

File: ckanext/data_qld/templates/scheming/snippets/display_field.html [hunk 1]
```diff
@@ -30,6 +30,6 @@
   {%- if field.field_name.lower() == 'url' and field.label.lower() == 'source' -%}
     Not applicable
   {%- else -%}
-    Field name '{{field.field_name}}' not in data
+    [blank]
```
Comment (ThrawnCA) [inline ckanext/data_qld/templates/scheming/snippets/display_field.html:33]: There is still a scenario test expecting the old message syntax.

**Review by ThrawnCA — CHANGES_REQUESTED** [review]

File: ckanext/data_qld/templates/scheming/snippets/display_field.html (hunk 1 above)
Comment (duttonw) [reply ckanext/data_qld/templates/scheming/snippets/display_field.html:33]: not surprising. I'm unsure we we even need to be 'emitting'  that not in data message, it should just skip that row.

**Review by ThrawnCA — APPROVED** [review]

**Comment by duttonw** [comment]
@ThrawnCA  not much left since the other branch was merged, only updated tests.

**Comment by ThrawnCA** [comment]
> not much left since the other branch was merged, only updated tests.

Those tests were important for verifying that CHDEXCHAE-60 is done, though.

**Pull request merged**